hexa/datasets/schema/types.py

```diff
@@ -217,6 +218,16 @@ def resolve_upload_url(obj, info, **kwargs):
         return None
 
 
+@dataset_version_file_object.field("fileMetadata")
+def resolve_version_file_metadata(obj: DatasetVersionFile, info, **kwargs):
+    try:
+        file_metadata = DatasetFileMetadata.objects.get(dataset_version_file=obj)
```

Comment:
You can have plenty of DatasetFileMetadata per DatasetVersionFile (as specified by your FK)

Reply:
I'd create a property on DatasetVersionFile to get the latest DatasetFileMetadata